models/intermediate/int_klaviyo__event_attribution.sql

```diff
@@ -5,7 +5,9 @@
         partition_by={
             "field": "occurred_on",
             "data_type": "date"
-        } if target.type != 'spark' else ['occurred_on']
+        } if target.type == 'bigquery' else none,
```

Comment:
Can you please document this in our Core processes doc? Would love to have this outlined & explained as I'm sure this took you quite a chunk of time to figure out this config.